src/parser/ml/FStar_Parser_LexFStar.ml

```diff
@@ -502,30 +506,35 @@ let rec token = lexer
       * creates a lexing conflict with op_infix3 which is caught below. *)
      one_line_comment (L.lexeme lexbuf) lexbuf
 
- | '"' -> string (Buffer.create 0) lexbuf.Ulexing.start_p lexbuf
+ | '"' -> string (Buffer.create 0) lexbuf.Sedlexing.start_p lexbuf
 
- | '`' '`' (([^'`' 10 13 0x2028 0x2029] | '`' [^'`' 10 13 0x2028 0x2029])+) '`' '`' ->
+ | '`', '`', (Plus (Compl ('`' | 10 | 13 | 0x2028 | 0x2029) | '`', Compl ('`' | 10 | 13 | 0x2028 | 0x2029))), '`', '`' ->
    IDENT (trim_both lexbuf 2 2)
 
- | op_token  -> L.lexeme lexbuf |> Hashtbl.find operators
+ | op_token_1
+ | op_token_2
+ | op_token_3
+ | op_token_4
+ | op_token_5 -> L.lexeme lexbuf |> Hashtbl.find operators
```

Comment:
Using a single large regexp is less efficient than the disjunction of five smaller ones?